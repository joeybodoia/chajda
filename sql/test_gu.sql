CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."');
