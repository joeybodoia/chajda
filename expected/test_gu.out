CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                 spacy_lemmatize                                                  
------------------------------------------------------------------------------------------------------------------
   મારું:3 ઉદાહરણ:4 પરીક્ષણ:5 વાક્ય:6 હું:10 પરીક્ષણનાં:11 કેસો:12 પેદા:13 કરવા:14 google:16 અનુવાદમાં:17 મૂકી:18 રહ્યો:19 છું:20
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                              spacy_lemmatize                               
----------------------------------------------------------------------------
   મારું ઉદાહરણ પરીક્ષણ વાક્ય હું પરીક્ષણનાં કેસો પેદા કરવા google અનુવાદમાં મૂકી રહ્યો છું
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   આ:2 મારું:3 ઉદાહરણ:4 પરીક્ષણ:5 વાક્ય:6 છે:7 કે:8 જે:9 હું:10 પરીક્ષણનાં:11 કેસો:12 પેદા:13 કરવા:14 માટે:15 google:16 અનુવાદમાં:17 મૂકી:18 રહ્યો:19 છું:20
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                    spacy_lemmatize                                     
----------------------------------------------------------------------------------------
   આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે google અનુવાદમાં મૂકી રહ્યો છું
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                      spacy_lemmatize                                                       
----------------------------------------------------------------------------------------------------------------------------
   ":2 મારું:4 ઉદાહરણ:5 પરીક્ષણ:6 વાક્ય:7 હું:11 પરીક્ષણનાં:12 કેસો:13 પેદા:14 કરવા:15 google:17 અનુવાદમાં:18 મૂકી:19 રહ્યો:20 છું.:21 ":22
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                 spacy_lemmatize                                 
---------------------------------------------------------------------------------
   " મારું ઉદાહરણ પરીક્ષણ વાક્ય હું પરીક્ષણનાં કેસો પેદા કરવા google અનુવાદમાં મૂકી રહ્યો છું. "
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                  spacy_lemmatize                                                                   
----------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 આ:3 મારું:4 ઉદાહરણ:5 પરીક્ષણ:6 વાક્ય:7 છે:8 કે:9 જે:10 હું:11 પરીક્ષણનાં:12 કેસો:13 પેદા:14 કરવા:15 માટે:16 google:17 અનુવાદમાં:18 મૂકી:19 રહ્યો:20 છું.:21 ":22
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                       spacy_lemmatize                                       
---------------------------------------------------------------------------------------------
   " આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે google અનુવાદમાં મૂકી રહ્યો છું. "
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                 spacy_lemmatize                                                  
------------------------------------------------------------------------------------------------------------------
   મારું:3 ઉદાહરણ:4 પરીક્ષણ:5 વાક્ય:6 હું:10 પરીક્ષણનાં:11 કેસો:12 પેદા:13 કરવા:14 Google:16 અનુવાદમાં:17 મૂકી:18 રહ્યો:19 છું:20
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                              spacy_lemmatize                               
----------------------------------------------------------------------------
   મારું ઉદાહરણ પરીક્ષણ વાક્ય હું પરીક્ષણનાં કેસો પેદા કરવા Google અનુવાદમાં મૂકી રહ્યો છું
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                             
-----------------------------------------------------------------------------------------------------------------------------------------
   આ:2 મારું:3 ઉદાહરણ:4 પરીક્ષણ:5 વાક્ય:6 છે:7 કે:8 જે:9 હું:10 પરીક્ષણનાં:11 કેસો:12 પેદા:13 કરવા:14 માટે:15 Google:16 અનુવાદમાં:17 મૂકી:18 રહ્યો:19 છું:20
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                    spacy_lemmatize                                     
----------------------------------------------------------------------------------------
   આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                      spacy_lemmatize                                                       
----------------------------------------------------------------------------------------------------------------------------
   ":2 મારું:4 ઉદાહરણ:5 પરીક્ષણ:6 વાક્ય:7 હું:11 પરીક્ષણનાં:12 કેસો:13 પેદા:14 કરવા:15 Google:17 અનુવાદમાં:18 મૂકી:19 રહ્યો:20 છું.:21 ":22
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                 spacy_lemmatize                                 
---------------------------------------------------------------------------------
   " મારું ઉદાહરણ પરીક્ષણ વાક્ય હું પરીક્ષણનાં કેસો પેદા કરવા Google અનુવાદમાં મૂકી રહ્યો છું. "
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                  spacy_lemmatize                                                                   
----------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 આ:3 મારું:4 ઉદાહરણ:5 પરીક્ષણ:6 વાક્ય:7 છે:8 કે:9 જે:10 હું:11 પરીક્ષણનાં:12 કેસો:13 પેદા:14 કરવા:15 માટે:16 Google:17 અનુવાદમાં:18 મૂકી:19 રહ્યો:20 છું.:21 ":22
(1 row)

SELECT spacy_lemmatize('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                       spacy_lemmatize                                       
---------------------------------------------------------------------------------------------
   " આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું. "
(1 row)

SELECT spacy_tsvector('gu',' "આ મારું ઉદાહરણ પરીક્ષણ વાક્ય છે કે જે હું પરીક્ષણનાં કેસો પેદા કરવા માટે Google અનુવાદમાં મૂકી રહ્યો છું."');
                                                               spacy_tsvector                                                               
--------------------------------------------------------------------------------------------------------------------------------------------
 'google':16 'અનુવાદમાં':17 'ઉદાહરણ':4 'કરવા':14 'કેસો':12 'છું':20 'પરીક્ષણ':5 'પરીક્ષણનાં':11 'પેદા':13 'મારું':3 'મૂકી':18 'રહ્યો':19 'વાક્ય':6 'હું':10
(1 row)

